python/examples/notebooks/optaa/process_ooinet_optaa.ipynb

```diff
@@ -0,0 +1,1203 @@
+      "text/plain": [
+       "<Figure size 1152x648 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "# Plot the instrument and co-located CTD temperature records\n",
+    "plt.rcParams.update({'font.size': 14})\n",
+    "fig = plt.figure(figsize=(16,9))\n",
+    "plt.plot(optaa['time'], optaa['temp'], '-k', label='CTD')\n",
+    "plt.plot(optaa['time'], optaa['internal_temp'], label='AC-S Internal')\n",
+    "plt.plot(optaa['time'], optaa['external_temp'], label='AC-S External')\n",
+    "plt.ylabel('Seawater Temperature (degC)')\n",
+    "plt.legend()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The CTD seawater temperature and the instrument internal temperature are inputs to the functions used to calculate the optical absorption and beam attenuation. The data from the external temperature sensor on the AC-S is not used, but it is should be producing values very close to the CTD (the AC-S temperaure sensor is not as accurate as the CTD). All three measurements should be very similar. If they are not, that would be a good indication that there is a problem with one of the sensors."
```

Comment:
Remove "is" from " the phrase "but it is should be producing values very close to the CTD"